.github/workflows/cicd-main.yml

```diff
@@ -115,8 +120,9 @@ jobs:
         ### \'\'
   
   L0_Unit_Tests_GPU:
-    needs: [cicd-test-container-setup]
+    needs: [cicd-test-container-setup, pre-flight]
```

Comment:
Thinking whether we need to do this
needs: [cicd-test-container-setup, pre-flight]
for each test or it's better to just make
cicd-test-container-setup depend on pre-flight
and not have all the rest of tests depend on pre-flight